lib/src/network/data/daily_quiz/daily_quiz_reference.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:safebump/src/network/firebase/base_collection.dart';
+import 'package:safebump/src/network/model/common/result.dart';
+import 'package:safebump/src/network/model/daily_quiz.dart/daily_quiz.dart';
+import 'package:safebump/src/utils/utils.dart';
+
+class DailyQuizReference extends BaseCollectionReference<DailyQuiz> {
+  DailyQuizReference()
+      : super(
+          FirebaseFirestore.instance
+              .collection('daily_quiz')
```

Comment:
With collection name, we should separate a class. should apply this for all place using firebase in the project. You can refactor that with other pull request
Example: `XCollection.dailyQuiz`